history.txt

```diff
@@ -1,6 +1,21 @@
 OMERO version history
 =====================
 
+5.1.0-m5 (March 2015)
+---------------------
+
+Developer preview release - **only intended as a developer preview for
+updating code before the full public release of 5.1.0. Use at your own risk**.
+
+Changes include:
+
+-  implementation of OMERO.mail for emailing users via the server
```

Comment:
I was going to put the cross-links in the What's New pages rather than here - if we add links here now they will have broken links until your docs are merged. Also, the only links we have on the history page at present are to the security vulnerabilities.